src/udisksspawnedjob.c

```diff
@@ -963,8 +976,6 @@ void udisks_spawned_job_start (UDisksSpawnedJob *job)
 
   if (job->child_stdin_fd != -1)
     {
-      job->input_string_cursor = job->input_string;
```

Comment:
Technically, this shouldn't be necessary, because it is done on setting `udisks_spawned_job_set_property(PROP_INPUT_STRING)` and a job will never be started twice (or will it?). But maybe you're right and it is cleaner to set it here. Should I change?